### ISSUE TITLE
Bug --> Admin/Settings/Overview : Open Updater does nothing

### ISSUE THREAD

**Issue by HuguesDug** [nextcloud/server#29692]
**Nextcloud version :** 22.2.0
**PHP version :** 7.3.31
**MySQL version :** 10.3.31
**OS :** Raspberry PI OS / Linux 5.10.63
**HTTPD :** nginx 1.14.2
**Browser :** any on windows 10

**Expected result:**
When a new release is available, update is offered in Admin/Settings/Overview.
The button "open updater" is diplayed
When clicking the button, the updater should show up and start the upgrade process.

**Current behavior:**
Open updater button is displayed
Clicking on the button generates no reaction from Nextcloud.
Problem has been there since several updates (21.0.0) if I remember correct.

**Work around:**
Update from the console via updater.phar that will complete without any problem.

**Logs and messages:**
No message displayed in nextcloud log
No message displayed in nginx log
However, while inspecting the webpage when clicking the button, 3 messages are displayed in the console.

-----------
_Warning_
globals.js:62 $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
ue @ globals.js:62
get @ globals.js:93
clickUpdaterButton @ UpdateNotification.vue:378
Qt @ vue.runtime.esm.js:1863
n @ vue.runtime.esm.js:2188
o._wrapper @ vue.runtime.esm.js:6961

_Error_
jquery.js:9600 GET https://dugres.hd.free.fr/nextcloud/apps/updatenotification/credentials 412
send @ jquery.js:9600
ajax @ jquery.js:9206
e.ajax.e.ajax @ jquery-migrate.min.js:2
clickUpdaterButton @ UpdateNotification.vue:378
Qt @ vue.runtime.esm.js:1863
n @ vue.runtime.esm.js:2188
o._wrapper @ vue.runtime.esm.js:6961

_Warning_
util.js:135 OC.Util.relativeModifiedDate is deprecated and will be removed in Nextcloud 21. See @nextcloud/moment

------------

**Comment by shufflerror** [comment]
Nextcloud version : 22.2.0
PHP version : 7.4.25
MySQL version : 10.5.13
OS : Debian 9.13

Same behavior as @HuguesDug on my two instances

**Comment by HuguesDug** [comment]
Solution found:
The documentation was changed for NGINX configuration.
https://docs.nextcloud.com/server/22/admin_manual/installation/nginx.html

I copied the new conf file and now it works.
